File: PhaseGateTwo/java/EvenDigits.java
public class EvenDigits{
	public static void main(String... args){


	

	for(int number = 1000; number <= 3000; number++){

	int firstNumber = number % 10;
	int firstNumberExtraction = number / 10;
	int secondNumber  = firstNumberExtraction % 10;
	int secondNumberExtraction = firstNumberExtraction / 10;
	int thirdNumber = secondNumberExtraction % 10;
	int thirdNumberExtraction = secondNumberExtraction / 10;
	int fourthNumber = thirdNumberExtraction % 10;

	if(firstNumber % 2 == 0 && secondNumber % 2 == 0 && thirdNumber % 2 == 0 && fourthNumber % 2 == 0) System.out.print(number + ",");


		}

	
	}

}

File: PhaseGateTwo/evendigits.py
numbers = []

for number in range(1000, 3001):
		
	first_number = number % 10
	first_number_extraction = number // 10
	second_number = first_number_extraction % 10
	second_number_extraction = first_number_extraction // 10
	third_number = second_number_extraction % 10
	third_number_extraction = second_number_extraction // 10
	fourth_number = third_number_extraction % 10

	if(first_number  % 2 == 0 and second_number  % 2 == 0 and third_number  % 2 == 0 and fourth_number % 2 == 0):
		numbers.append(number)
print(numbers)
		
	
	
	



